src/zenml/zen_stores/migrations/versions/6917bce75069_add_pipeline_run_unique_constraint.py

```diff
@@ -0,0 +1,65 @@
+"""Add pipeline run unique constraint [6917bce75069].
+
+Revision ID: 6917bce75069
+Revises: 0.47.0
+Create Date: 2023-11-15 16:07:24.343126
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "6917bce75069"
+down_revision = "0.47.0"
+branch_labels = None
+depends_on = None
+
+
+def add_orchestrator_run_id_for_old_runs() -> None:
```

Comment:
No `baris_number` column? :(

Reply:
Decided to add ugly values to existing columns instead of creating a new column just to add ugly values 😄

Reply:
Btw I added some notes on this migration in the PR description, do you think this is an acceptable risk or do we have to find a way around that potential issue?

Reply:
So the main risk is that there might be users with custom orchestrators where the `orchestrator_run_id` is not unique? I would say that's an acceptable risk as long as it does not happen with any of our official orchestrators.

That said, I suppose we could handle this case in the migration if we wanted by first searching for rows that violate the constraints and abandoning the migration with a corresponding error message if there are duplicates. Whether it's worth implementing this is another question - I'll leave that up to you 😉

Reply:
I like that idea, better to be safe with migrations. Shouldn't be too difficult to implement either, I'll give it a try.

Reply:
Done